<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pathway</class>
 <widget class="QMainWindow" name="pathway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pathway</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 221, 221);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>551</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);
border:0px;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="footGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>891</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="leftGroupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>191</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="peopleTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:0px;</string>
     </property>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主机名</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="peopleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;幼圆&quot;;
color: rgb(167, 167, 167);</string>
     </property>
     <property name="text">
      <string>附近的人</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>206</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(229, 229, 229);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rightGroupBox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>161</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>551</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 254, 254);</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>Pathway</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="minPushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
